templates/common/essentials/config/endpoints.js

```diff
@@ -14,4 +14,3 @@ export const endpoints = {
  */
 export const contentDomain = 'https://jsonplaceholder.typicode.com/';
```

Comment:
This config should also be a package under root/package/config

Reply:
Do you mean endpoints.js should go in root/packages/config or the whole common/essentials/config folder?